django_filters/views.py

```diff
@@ -36,7 +36,10 @@ def get_filterset_kwargs(self, filterset_class):
         """
         Returns the keyword arguments for instanciating the filterset.
         """
-        kwargs = {'data': self.request.GET or None}
+        kwargs = {
+            'data': self.request.GET or None,
+            'request': self.request,
```

Comment:
No. I want `FilterSet.__init__` to explicitly take the `data` kwarg. I know it can be referenced from the request but I see the latter as optional in a way that the data is not...